### PR TITLE
Remove POD

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

Remove all references to the POD, and change the Create Loan flow to mint the collateral NFT in the app itself. Ignore private Loan metadata for now and only pin public data to IPFS.

[#2154](https://github.com/centrifuge/apps/issues/2154)

### Approvals

- [ ] Dev

### CODE REVIEW

File: centrifuge-app/src/utils/useDAOConfig.tsx [hunk 1]
```diff
@@ -23,15 +22,12 @@ export const useDAOConfig = () => {
   const query = useQuery(
     'daoData',
     async () => {
-      const res = await fetch(
```
Comment (hieronx) [inline centrifuge-app/src/utils/useDAOConfig.tsx:26]: Why was this changed?

File: centrifuge-app/src/pages/IssuerPool/Assets/CreateLoan.tsx [hunk 2]
```diff
@@ -168,30 +167,44 @@ function IssuerCreateLoan() {
...
     (cent) =>
-      ([submittable], options) =>
-        cent.getApi().pipe(switchMap((api) => cent.wrapSignAndSend(api, submittable, { ...options, sendOnly: true }))),
+      (
+        [collectionId, nftId, owner, metadataUri, pricingInfo]: [string, string, string, string, LoanInfoInput],
+        options
+      ) => {
+        return centrifuge.pools.createLoan([pid, collectionId, nftId, pricingInfo], { batch: true }).pipe(
+          switchMap((createTx) => {
+            const tx = api.tx.utility.batchAll([
```
Comment (hieronx) [inline centrifuge-app/src/pages/IssuerPool/Assets/CreateLoan.tsx:192]: Really nice this is now 1 batch!!

**Review by hieronx — APPROVED** [review]
Did some testing on dev, it works perfectly, really nice work! 💯

File: centrifuge-app/src/utils/useDAOConfig.tsx (hunk 1 above)
Comment (onnovisser) [reply centrifuge-app/src/utils/useDAOConfig.tsx:26]: ah oops
